.circleci/config.yml

```diff
@@ -22,6 +22,13 @@ jobs:
       - checkout
       - run:
           command: git submodule update --init --recursive
+      - run:
+        #Circle is bad about setting up your remotes for you, you must use origin/
+          command: |
+            if [ $(git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep api/ | wc -l | xargs) == 0 ] && [ ${CIRCLE_BRANCH} != "master" ]; then
```

Comment:
I would reverse the order of these predicates. I would also include the check for empty CIRCLE_BRANCH (I believe this happens when we tag a release - not 100%)

Reply:
Will CIRCLE_BRANCH being empty be caught by != master?

Reply:
Will do!